pkg/persistence/object_store_layout.go

```diff
@@ -98,3 +98,12 @@ func (l *ObjectStoreLayout) getRestoreLogKey(restore string) string {
 func (l *ObjectStoreLayout) getRestoreResultsKey(restore string) string {
 	return path.Join(l.subdirs["restores"], restore, fmt.Sprintf("restore-%s-results.gz", restore))
 }
+
+// TODO: Add keys for VS & VSContents json.gz files
+func (l *ObjectStoreLayout) getCSIVolumeSnapshotContentsKey(backup string) string {
```

Comment:
is this supposed to be just `getCSIVolumeSnapshotsKey`? (no "Contents")

Reply:
Yep, bad copy/paste